packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -536,7 +553,7 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
       renderer
     );
 
-    vtkCamera.setIsPerformingCoordinateTransformation(false);
+    vtkCamera.setIsPerformingCoordinateTransformation?.(false);
```

Comment:
I must be missing something, but I am trying to understand how `setIsPerformingCoordinateTransformation` is ever null such that the `?.` is needed. Plesase help me here - what am I missing? :)

Reply:
It is a custom method we add to our custom camera, so it might be possible that the other camera (vtkCamera) is used and hence we don't have it